pom.xml

```diff
@@ -37,17 +37,17 @@
 
     <properties>
         <!--  default URL properties -->
-        <codice.scm.connection.url />
-        <snapshots.repository.url />
-        <reports.repository.url />
+        <codice.scm.connection.url/>
+        <snapshots.repository.url/>
+        <reports.repository.url/>
 
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
         <codice-maven.version>0.1</codice-maven.version>
         <codice-test.version>0.2</codice-test.version>
 
-        <guava.version>27.0-jre</guava.version>
+        <guava.version>30.0-android</guava.version>
```

Comment:
Definitely don't want this. If we want to upgrade we need the JRE flavor of `30.0`, see:
https://github.com/google/guava/releases/tag/v30.0